github/checks.go

```diff
@@ -291,10 +291,10 @@ func (s *ChecksService) ListCheckRunsCheckSuite(ctx context.Context, owner, repo
 	return checkRunResults, resp, nil
 }
 
-// ListCheckSuiteOptions represents  parameters to list check suites.
+// ListCheckSuiteOptions represents parameters to list check suites.
 type ListCheckSuiteOptions struct {
 	CheckName *string `url:"check_name,omitempty"` // Filters checks suites by the name of the check run.
-	Filter    *string `url:"filter,omitempty"`     // Filters check suites by GitHub App id.
+	AppID     *int    `url:"app_id,omitempty"`     // Filters check suites by GitHub App id.
```

Comment:
I'm confused by this change.
I'm looking at this:
https://developer.github.com/v3/checks/runs/#list-check-runs-for-a-specific-ref
and it appears to take the `filter` argument, not `app_id`.

Reply:
@gmlewis This one is for `check suites` :) . https://developer.github.com/v3/checks/suites/#list-check-suites-for-a-specific-ref